### PR TITLE
Add DIN Rail Mount for FYSETC Spider

### PR DESCRIPTION
**Pull request by hoschi73**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

DIN Rail Mount for FYSETC Spider based on the Voron2 Din-Rail-Bracket
 
#### How was it tested? 

All Parts were printed and testfitted (both versions).

### CODE REVIEW

File: printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+for horizontal or vertical mounting
+
+![FYSETC_Spider_DIN_Mount](./IMG/main.jpg)
+
+Required material:
+
+2x printed DIN mounts
+
+for horizontal mounting:
+spider_bracket_horizontal_terminal.stl
```
Comment (KiloQubit) [inline printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md:12]: Please add `<br/>` after these lines so they don't appear on a single line.  Alternatively use the list syntax https://www.markdownguide.org/basic-syntax/#unordered-lists

File: printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+spider_bracket_horizontal_usb.stl
+
+for vertical mountin:
```
Comment (KiloQubit) [inline printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md:15]: Replace `mountin` with `mounting`.  Either add `<br/>` at the end of these lines or use the list syntax as above.

File: printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+FYSETC Spider DIN Rail Mount
+
+for horizontal or vertical mounting
+
+![FYSETC_Spider_DIN_Mount](./IMG/main.jpg)
+
+Required material:
```
Comment (KiloQubit) [inline printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md:7]: Replace `material` with `materials`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  Please check some changes below.

- Please delete `IMG/horizontal_rail.jpg` and `IMG/vertical.rail.jpg`, since they're already in `IMG/main.jpg`
- You have two copies of `vertical.rail.jpg` - one in `printer_mods/hoschi/FYSETC_Spider_DIN_Mount/IMG/vertical.rail.jpg` and one in `/vertical.rail.jpg` - please remove `/vertical.rail.jpg` as well (the one at the top level)
- Please remove `printer_mods/hoschi/FYSETC_Spider_DIN_Mount/IMG/.del`

File: printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
+Required materials:
 
 for horizontal mounting:
-spider_bracket_horizontal_terminal.stl
-spider_bracket_horizontal_usb.stl
 
-for vertical mountin:
-spider_bracket_vertical_terminal.stl
-spider_bracket_vertical_usb.stl
+- 1x printed bracket terminal side of the Spider Board: ![spider_bracket_horizontal_terminal.stl](./STL/spider_bracket_horizontal_terminal.stl)
```
Comment (KiloQubit) [inline printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md:11]: For simple links, don't put `!` in front of it - this makes it think it's an image, it's not, so it's broken.  Just

    [spider_bracket_horizontal_terminal.stl](./STL/spider_bracket_horizontal_terminal.stl)

should work.

Screen Shot 2021-05-12 at 19 43 19

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates.  There are few more things to fix.  The links you tried to add to the README are actually image references, and they're broken.

The jpg file in `/` still hasn't been deleted:
Screen Shot 2021-05-12 at 19 39 12

**Comment by hoschi73** [comment]
i hope i got it this time. thanks you for your input and patience.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Comment by hoschi73** [comment]
Hi,
i have a minor update to the mod. (With the original spacers the pins from the power terminals were digging into the mount. Spacer height extended by 1.5 mm).

How can i submit the changes?

Regards 
Sascha

**Pull request merged**